docs/weave-tutorial/08-app.md

```diff
@@ -131,18 +133,18 @@ func Chain(authFn x.Authenticator, minFee coin.Coin) app.Decorators {
 - `utils.NewKeyLogging` enables logging in application
 - `utils.NewRecovery`: Recovery is a decorator to recover from panics in transactions, so we can log them as errors
 - `utils.NewKeyTagger`:
-  - **KeyTagger** is a decorate that records all _Set/Delete_ operations performed by it's children and adds all those keys as DeliverTx tags
-  - Tags is the hex encoded key, value is **s** (for set) or *d* (for delete)
+  - **KeyTagger** is a decorater that records all _Set/Delete_ operations performed by its children and adds all those keys as DeliverTx tags
```

Comment:
`  - **KeyTagger** is a decorator that records all _Set/Delete_ operations performed by its children and adds all those keys as DeliverTx tags
`